include/sway/tree/container.h

```diff
@@ -72,9 +72,13 @@ struct sway_container {
 	// For C_OUTPUT, this is the output position in layout coordinates
 	// For other types, this is the position in output-local coordinates
 	double x, y;
+	double saved_x, saved_y;
```

Comment:
I haven't really finished reading all of this but instead of saving the x/y it's probably better to just call arrange_windows again

Reply:
You're right in the tiling case, but It might be needed for the floating case.

Reply:
Fair point. But in the tiling case we cannot assume that these coordinates will continue to be valid after unfullscreening.

Reply:
I'd already removed these variables before seeing the latest two comments. I think it's best to leave them out for now and look at it once floating is implemented.